server/index.js

```diff
@@ -0,0 +1,35 @@
+import http from 'http'
+import { URL } from 'url'
+
+import getBrowsers from './get-browsers.js'
+
+const DEFAULT_QUERY = 'defaults'
+const PORT = process.env.PORT || 5000
+
+http
+  .createServer(async (req, res) => {
+    let url = new URL(req.url, `http://${req.headers.host}/`)
+
+    if (url.pathname === '/') {
+      let query = url.searchParams.get('q') || DEFAULT_QUERY
+      let queryWithoutQuotes = query.replace(/'/g, '')
+
+      try {
+        let browsers = await getBrowsers(queryWithoutQuotes)
+        res.writeHead(200, {
+          'Access-Control-Allow-Origin': '*',
+          'Content-Type': 'text/json'
+        })
+        res.write(JSON.stringify(browsers))
+        res.end()
+      } catch (error) {
+        res.writeHead(400, {
+          'Access-Control-Allow-Origin': '*',
+          'Content-Type': 'text/json'
+        })
+        res.write(JSON.stringify({ error }))
```

Comment:
Error is an object. It will be stringified like `error: {}`

```suggestion
        res.write(error.message)
```

Reply:
Good. Let the error be a string for now.

_I had the idea of an object {type, message}, but I haven't come up with other types yet. Especially after all the errors in the Browserslist seem to be handled (about my last browserslist PR)._